app/public/css/styles.css

```diff
@@ -3775,6 +3775,23 @@ input:checked + .slider:before {
   margin-bottom: 32px;
 }
 
+/* MULTI ASSETS SCREEN: START */
+.mobile-multi-assets-page-wrapper {
+  margin-top: 32px;
+  margin-bottom: 32px;
+}
+
+.multi-assets-page-list {
+  max-height: 300px;
+  overflow: auto;
+}
+
+.multi-asset-page-policy {
+  /* TODO: rethink, larger width causes cards on the left to shrink too much */
```

Comment:
I was thinking about this issue as well. Maybe we can remove flex grow/shrink from the columns?